src/Feature/Navigation/client/Header/NavMenu/index.tsx

```diff
@@ -1,57 +1,61 @@
 //    Copyright 2020 EPAM Systems, Inc.
-// 
+//
 //    Licensed under the Apache License, Version 2.0 (the "License");
 //    you may not use this file except in compliance with the License.
 //    You may obtain a copy of the License at
-// 
+//
 //      http://www.apache.org/licenses/LICENSE-2.0
-// 
+//
 //    Unless required by applicable law or agreed to in writing, software
 //    distributed under the License is distributed on an "AS IS" BASIS,
 //    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
 
-import { Image } from '@sitecore-jss/sitecore-jss-react';
+import * as JSS from 'Foundation/ReactJss/client';
 import * as React from 'react';
 
-import classNames from 'classnames';
+import classnames from 'classnames';
 
-import * as JSS from 'Foundation/ReactJss/client';
+import { Image } from '@sitecore-jss/sitecore-jss-react';
 import { NavigationLink } from 'Foundation/UI/client';
 
-import { NavMainProps, NavMainState } from './models';
+import { NavBarProps, NavBarState } from './models';
 import './styles.scss';
 
-export default class NavMain extends JSS.SafePureComponent<NavMainProps, NavMainState> {
-
+export class NavMenuComponent extends JSS.SafePureComponent<NavBarProps, NavBarState> {
   protected dropMenu(e: React.MouseEvent<HTMLElement>) {
     const elem = e.target as HTMLElement;
     e.preventDefault();
     elem.nextElementSibling.classList.toggle('drop-open');
     elem.classList.toggle('i-open');
-}
+  }
   protected safeRender() {
-    const { menuItems } = this.props;
+    const { menuItems } = this.props.fields.data.datasource;
 
     return (
-      <nav id="nav-main">
+      <nav className="nav-bar">
```

Comment:
Why?

Reply:
Because navigation menu not unique thing and we don't use id out of css